mobile/test/common/BUILD

```diff
@@ -18,6 +18,7 @@ envoy_cc_test(
 envoy_cc_test(
     name = "internal_engine_test",
     srcs = ["internal_engine_test.cc"],
+    rbe_pool = "2core",
```

Comment:
mobile CI doesnt use these pools

for ref here are the mobile pools:

https://envoy.cluster.engflow.com/restatus

and the envoy ones:

https://mordenite.cluster.engflow.com/restatus

Reply:
also mobile is not set up to use `rbe_pool` so it will either break stuff or at best be ignored

it is, however, OOMing

```console
ERROR: /source/mobile/test/common/BUILD:18:14: Compiling test/common/internal_engine_test.cc failed: (Killed): clang-14 failed: error executing command (from target //test/common:internal_engine_test) 
```

which means there is a bigger issue in terms of mobile workers and resources - probably you will need to raise this on the related channel